src/op_mode/ipsec.py

```diff
@@ -438,6 +463,49 @@ def _get_all_sitetosite_peers_name_list() -> list:
     return peers_list
 
 
+def initiate_peer_with_childsas(peer: str,
+                                tunnel: typing.Optional[str] = None):
+    tunnel_sw = None
+    if tunnel:
+        if tunnel.isnumeric():
+            tunnel_sw = f'{peer}-tunnel-{tunnel}'
+        elif tunnel == 'vti':
+            tunnel_sw = f'{peer}-vti'
+    try:
+        con_list: list = vyos.ipsec.get_vici_connection_by_name(peer)
+        if not con_list:
+            raise vyos.opmode.IncorrectValue(
+                f'Peer\'s {peer} SA(s) not loaded, aborting')
+
+        childsa_name_list: list = _get_con_childsa_name_list(con_list, {
+            'close_action': b'start'})
+        if not childsa_name_list:
+            if tunnel:
+                raise vyos.opmode.IncorrectValue(
+                    f"Peer {peer} tunnel {tunnel} SA is not configured with 'close_action=start', aborting")
+            else:
+                raise vyos.opmode.IncorrectValue(
+                    f"Peer {peer} SAs with 'close_action=start' not found, aborting")
+        if not tunnel_sw:
+            vyos.ipsec.vici_initiate_all_child_sa_by_ike(peer,
+                                                         childsa_name_list)
+            print(f'Peer {peer} initiate result: success')
```

Comment:
I wonder if printing to stdout is worthwhile here. Maybe just remove those prints.

Reply:
I added Information prints when a task is completed. 
Because this command consists of two tasks - termination and initialization.
Do you think this message is not necessary?